src/main/kotlin/io.javalin/core/util/Util.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Javalin - https://javalin.io
+ * Copyright 2017 David Åse
+ * Licensed under Apache 2.0: https://github.com/tipsy/javalin/blob/master/LICENSE
+ */
+
+package io.javalin.core.util
+
+import io.javalin.HaltException
+import org.slf4j.LoggerFactory
+import java.io.IOException
+import java.io.InputStream
+import java.io.OutputStream
+import java.util.*
+
+object Util {
+
+    private val log = LoggerFactory.getLogger(Util::class.java)
+
+    fun notNull(obj: Any?, message: String) {
+        if (obj == null) {
+            throw IllegalArgumentException(message)
+        }
+    }
+
+    fun classExists(className: String): Boolean {
+        try {
+            Class.forName(className)
+            return true
+        } catch (e: ClassNotFoundException) {
+            return false
+        }
+
+    }
+
+    private val dependencyCheckCache = HashMap<String, Boolean>()
+
+    fun ensureDependencyPresent(dependencyName: String, className: String, url: String) {
+        if (dependencyCheckCache.getOrDefault(className, false)) {
+            return
+        }
+        if (!classExists(className)) {
+            val message = "Missing dependency '$dependencyName'. Please add dependency: https://mvnrepository.com/artifact/$url"
+            log.warn(message)
+            throw HaltException(500, message)
+        }
+        dependencyCheckCache[className] = true
+    }
+
+    fun pathToList(pathString: String): List<String> {
+        return pathString.split("/").filter { it.isNotEmpty() }.toList();
+    }
+
+    fun printHelpfulMessageIfLoggerIsMissing() {
+        if (!classExists("org.slf4j.impl.StaticLoggerBinder")) {
+            val message = """
+            "-------------------------------------------------------------------"
```

Comment:
It is not clear to me why you use a multiline string but then also starts and terminate your lines with "\""